apps/omg_utils/test/omg_utils/http_rpc/response_test.exs

```diff
@@ -23,7 +23,6 @@ defmodule OMG.Utils.HttpRPC.ResponseTest do
     txbytes: nil,
     txhash: nil,
     txindex: nil,
-    metadata: nil,
```

Comment:
this seems unrelated to this PR. is this intentional?

Reply:
yes, there are two metadata fields in the struct